fitbenchmarking/input_parsing.py

```diff
@@ -218,16 +218,20 @@ def load_neutron_data_fitting_problem_file(fname):
 
     @param fname :: name of the file to load
     """
+
     with open(fname) as probf:
+
         entries = get_neutron_data_problem_entries(probf)
-        # get the path to the data
```

Comment:
Please keep comment. In fact I would add to it explaining in code that data in data_files is will be added to the data search path